Makefile

```diff
@@ -118,3 +121,16 @@ dist: clean docs assets
 
 install: clean
 	python setup.py install
+
+pex:
+	ls dist/ka_lite-*.whl | while read whlfile; do pex $$whlfile -o dist/kalite-`unzip -p $$whlfile kalite/VERSION`.pex -m kalite --python-shebang=/usr/bin/python; done
```

Comment:
Instead of writing something in `kalite/VERSION` and then reading back the file through piped output from an unzip command, I would suggest that you just replace `unzip -p $$whlfile kalite/VERSION` with `git describe --tags` ;)